### PR TITLE
[Merged by Bors] - feat: Essential supremum according to the counting measure

### PR DESCRIPTION
**Pull request by YaelDillies**
... is just the usual supremum

From LeanAPAP

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by j-loreaux** [comment]
I'll let someone with more familiarity with the measure theory library make the final decision here.

maintainer merge

File: Mathlib/MeasureTheory/OuterMeasure/AE.lean [hunk 1]
```diff
@@ -106,15 +106,25 @@ theorem ae_ball_iff {ι : Type*} {S : Set ι} (hS : S.Countable) {p : α → ∀
     (∀ᵐ x ∂μ, ∀ i (hi : i ∈ S), p x i hi) ↔ ∀ i (hi : i ∈ S), ∀ᵐ x ∂μ, p x i hi :=
   eventually_countable_ball hS
 
-theorem ae_eq_refl (f : α → β) : f =ᵐ[μ] f :=
-  EventuallyEq.rfl
+lemma ae_eq_refl (f : α → β) : f =ᵐ[μ] f := EventuallyEq.rfl
+lemma ae_eq_rfl {f : α → β} : f =ᵐ[μ] f := EventuallyEq.rfl
```
Comment (semorrison) [inline Mathlib/MeasureTheory/OuterMeasure/AE.lean:110]: Why this duplication? Could one be deprecated?

**Comment by semorrison** [comment]
Please redo `maintainer-merge` afterwards.

File: Mathlib/MeasureTheory/OuterMeasure/AE.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/MeasureTheory/OuterMeasure/AE.lean:110]: We have this duplication for every concrete reflexive relation, eg `le_refl`/`le_rfl`. I'm not sure this PR is the place to upturn such a widespread pattern and naming convention.

**Comment by YaelDillies** [comment]
Like so?

maintainer merge

**Comment by b-mehta** [comment]
Removing my request for review here, since I don't know this part of the library well enough to comment.

File: Mathlib/MeasureTheory/OuterMeasure/AE.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/MeasureTheory/OuterMeasure/AE.lean:110]: :-( This makes me really sad, but okay.

**Comment by RemyDegenne** [comment]
Thanks!
bors r+

**Pull request closed without merging**